noxfile.py

```diff
@@ -44,6 +54,13 @@ def docs(session):
 @nox.session(name="docs-live", reuse_venv=True)
 def docs_live(session):
     _install_this_editable(session, extras=["docs"])
-    session.run(
-        "sphinx-autobuild", "-b", "html", "docs/", "build/docs", *session.posargs
-    )
+    cmd = [
+        "sphinx-autobuild",
+        "-b",
+        "html",
+        "docs/",
+        "build/docs",
+        "--port",
+        "0",
```

Comment:
Just a refactoring for readability. To put flags and the associated value on the same line is a practice I've come to appreciate.

```suggestion
        "-b=html",
        "--port=0",
        "docs/",
        "build/docs",
```

Reply:
ah nice - I didn't realize that the `=` still worked in this case. Black was clobbering the "two arguments per line" setup I had before.